src/script/util/StringUtil.ts

```diff
@@ -161,3 +161,20 @@ export const fixWebsocketString = (originalString: string): string => {
   const decoder = new TextDecoder();
   return decoder.decode(charArray);
 };
+
+const umlautMap: Record<string, string> = {
```

Comment:
im a bit confused by a few things here: 
It is called umlaut map but it seems to include a large number of accented/modified characters. An umlaut is just a germanic ¨ character. 

Additionally, it seems to be missing several additional possible umlauts:
ä ö ë ï 

I imagine we should be including any possible modified character right?

Reply:
fixed in https://github.com/wireapp/wire-webapp/pull/14634/commits/1495f6e3b82a46d0dc2a19da7c2e98393037d911